cl/favorites/utils.py

```diff
@@ -1,5 +1,6 @@
 from dataclasses import dataclass
 from datetime import timedelta
+from typing import Optional
```

Comment:
You don't need to import Optional for type hints in Python 3.10 and later. Python 3.10 introduced the union operator` | `, which you can read about in [PEP 604](https://www.python.org/dev/peps/pep-0604/).  For Python 3.10 and up, the preferred (and more concise) way to specify an optional argument is `Type | None`.  For example, use `str | None` or `list | None`.